src/unix/aix-common.c

```diff
@@ -268,6 +268,13 @@ int uv_interface_addresses(uv_interface_address_t** addresses,
     else
       address->netmask.netmask4 = *((struct sockaddr_in*) &p->ifr_addr);
 
+    if (flg.ifr_flags & IFF_BROADCAST && p->ifr_broadaddr != NULL) {
```

Comment:
style nit (if I recall my libuv style correctly):
```suggestion
    if (flg.ifr_flags & IFF_BROADCAST != 0 && p->ifr_broadaddr != NULL) {
```

Reply:
Done